tools/dromajo/dromajo.mk

```diff
@@ -49,7 +49,8 @@ ifdef ENABLE_DROMAJO
 EXTRA_SIM_FLAGS += $(DROMAJO_FLAGS)
 
 # CC flags needed for all simulations
-EXTRA_SIM_CC_FLAGS += -I$(DROMAJO_DIR)
+EXTRA_SIM_CFLAGS += -I$(DROMAJO_DIR)
+EXTRA_SIM_CXXFLAGS += -I$(DROMAJO_DIR)
```

Comment:
Do we actually have a use case for splitting these apart? Unless the answer is yes this seems like an anti-feature.